src/models/lamp.ts

```diff
@@ -157,6 +157,10 @@ export class Lamp {
     return true;
   }
 
+  private padLeftId(str: string) {
+    return '0'.repeat(5 - str.length) + str;
```

Comment:
@pikulsky According to MDN the .repeat() method is not available in the Android WebView (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat). This means that the app will crash on devices older than 4.4.

Reply:
However, these devices are rather old and not very common now, so I think this change is ok to merge.